packages/pds/src/app-view/api/app/bsky/graph/getFollowers.ts

```diff
@@ -43,7 +43,13 @@ export default function (server: Server, ctx: AppContext) {
         )
         .where(notSoftDeletedClause(ref('creator_repo')))
         .whereNotExists(
-          graphService.blockQb(requester, [ref('follow.subjectDid')]),
+          graphService.blockQb(requester, [ref('follow.creator')]),
+        )
+        .whereNotExists(
+          graphService.blockRefQb(
+            ref('follow.subjectDid'),
+            ref('follow.creator'),
+          ),
         )
```

Comment:
If this ends-up being too heady of a query, we can always use the same approach as `blocksForPosts()`/`getBlockSet()`.

Reply:
yeah i think we feel it out - i think this is a bit easier to model in-line than as a separate query (in comparison to replies). but if we're having trickery with it, we can totally split it out